### PR TITLE
Fix #1629 : replace story_text_size to reading_text_size

### LINKED ISSUE #1629: Update `StoryTextSize` to `ReadingTextSize`
Mock Link: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/2776cbb1-22b0-42c9-b70b-2136bdb361da/

Earlier in the app we were using `StoryTextSize` text in `OptionsActivity` as well as `ExplorationPlayer`. Now that text has been updated to `ReadingTextSize`.

This change should be made in all files, test cases, domain layer, proto layer, xml files, etc.

### PR DESCRIPTION
**Pull request by prayutsu**
## Explanation

Fixes #1629 : Replace all the occurrences of storyTextSize with readingTextSize

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

Reviewers - @anandwana001 @rt4914

### CODE REVIEW

**Comment by prayutsu** [comment]
@rt4914 Replaced all the occurrences of 'StoryTextSIze' with 'ReadingTextSize'

**Comment by prayutsu** [comment]
@rt4914 PTAL

File: app/src/main/java/org/oppia/app/options/OptionsReadingTextSizeViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package org.oppia.app.options
+
+import androidx.databinding.ObservableField
+
+/** StoryText size settings view model for the recycler view in [OptionsFragment]. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/options/OptionsReadingTextSizeViewModel.kt]: Update this comment too.

File: app/src/main/java/org/oppia/app/options/ReadingTextSizeActivity.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
+package org.oppia.app.options
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The activity to change the Text size of the Story content in the app. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/options/ReadingTextSizeActivity.kt]: Update to `/** The activity to change the text size of the reading content in the app. */`

File: app/src/main/java/org/oppia/app/options/ReadingTextSizeFragment.kt [hunk 3]
```diff
@@ -8,19 +8,19 @@ import android.view.ViewGroup
 import org.oppia.app.fragment.InjectableFragment
 import javax.inject.Inject
 
-private const val KEY_STORY_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE =
-  "STORY_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE"
+private const val KEY_READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE =
+  "READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE"
 
 /** The fragment to change the text size of the story content in the app. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/options/ReadingTextSizeFragment.kt]: Update this comment too.

File: app/src/main/java/org/oppia/app/player/exploration/ExplorationFragment.kt [hunk 4]
```diff
@@ -59,10 +59,10 @@ class ExplorationFragment : InjectableFragment() {
   override fun onAttach(context: Context) {
     super.onAttach(context)
     fragmentComponent.inject(this)
-    val storyTextSize =
+    val readingTextSize =
       arguments!!.getString(STORY_DEFAULT_FONT_SIZE_ARGUMENT_KEY)
-    checkNotNull(storyTextSize) { "ExplorationFragment must be created with a story text size" }
-    fontScaleConfigurationUtil.adjustFontScale(context, storyTextSize)
+    checkNotNull(readingTextSize) { "ExplorationFragment must be created with a story text size" }
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/player/exploration/ExplorationFragment.kt]: Update the message:

`ExplorationFragment must be created with a reading text size`

File: model/src/main/proto/profile.proto [hunk 5]
```diff
@@ -98,7 +98,7 @@ message ProfileChooserUiModel {
 }
 
 // Supported Story text sizes for user preference.
```
Comment (rt4914) [inline model/src/main/proto/profile.proto]: Update this to:
`// Supported reading text sizes for user preference.`

File: model/src/main/proto/profile.proto [hunk 6]
```diff
@@ -58,7 +58,7 @@ message Profile {
   int64 date_created_timestamp_ms = 9;
 
   // Represents user selected story-text-size.
```
Comment (rt4914) [inline model/src/main/proto/profile.proto]: Update this to:
`Represents user selected reading-text-size.`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Just few changes. Thanks.

There is one issue with branch name, the branch name should have all `small characters` separated by `underscore`. Refernece: https://github.com/oppia/oppia-android/pull/1685#pullrequestreview-472435495

**Comment by prayutsu** [comment]
> Just few changes. Thanks.
> 
> There is one issue with branch name, the branch name should have all `small characters` separated by `underscore`. Refernece: [#1685 (review)](https://github.com/oppia/oppia-android/pull/1685#pullrequestreview-472435495)

Sure I will fix those up. Thanks @rt4914

**Comment by prayutsu** [comment]
@rt4914 Do I need to change my branch name? Actually I changed it locally but it didn't change on my fork.

**Comment by aggarwalpulkit596** [comment]
You will most likely have to create another PR for that once you change your branch name it won't be reflected to this branch/PR

**Comment by prayutsu** [comment]
hey @rt4914, I just changed the branch name, actually, a new branch has been created. So, do I need to create another PR?

**Comment by prayutsu** [comment]
> You will most likely have to create another PR for that once you change your branch name it won't be reflected to this branch/PR

Oh, I already got the answer. Thanks to @aggarwalpulkit596

**Comment by rt4914** [comment]
> hey @rt4914, I just changed the branch name, actually, a new branch has been created. So, do I need to create another PR?

Yes, please create a PR. Once new PR is created, close this PR with appropriate message.

**Comment by prayutsu** [comment]
Created New PR https://github.com/oppia/oppia-android/pull/1696

**Comment by rt4914** [comment]
> Created New PR #1696

Thanks for mentioning this comment.

**Pull request closed without merging**